api/net/Areas/Services/Controllers/ContentController.cs

```diff
@@ -252,6 +252,24 @@ public async Task<IActionResult> UpdateAsync(ContentModel model, bool index = fa
 
         return new JsonResult(new ContentModel(content));
     }
+    /// <summary>
+    /// Update content for the specified 'id'.
+    /// Will not trigger any re-index or audit trail update
+    /// </summary>
+    /// <param name="model"></param>
+    /// <param name="newStatus">New ContentStatus.</param>
+    /// <returns></returns>
+    [HttpPut("{id}/status")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType(typeof(ContentModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
+    [SwaggerOperation(Tags = new[] { "Content" })]
+    public IActionResult UpdateStatusAsync(ContentModel model, ContentStatus newStatus)
```

Comment:
I would just include the content model.  Whatever status is in the model should be the new status.  But it can be either way.